src/Inventory/Asset/NetworkPort.php

```diff
@@ -280,6 +280,7 @@ private function handleMacConnection(\stdClass $port, int $netports_id)
         }
 
         $found_macs = [];
+        //FIXME Static analysis says this property is always an empty array. I don't see it get set even within the rule(s)
```

Comment:
As far as I can see; shit is set in the same file:
```php
$this->connection_ports[$itemtype][$ports_id] = $ports_id;
```

But I'm pretty sure this part may not work; I never had any data that suits those cases.